pages/share.jsx

```diff
@@ -2,9 +2,13 @@ import React  from 'react';
 import Footer from '../components/footer.jsx';
 import Header from '../components/header.jsx';
 import Social from '../components/social.jsx';
+import analytics from '../public/js/analytics.js';
 
 var ThankYou = React.createClass({
   mixins: [require('react-intl').IntlMixin],
+  componentDidMount: function() {
+    analytics();
```

Comment:
I am pretty sure we don't want this here.

1. It'll double count locales with share in the thank you page.
2. Share itself isn't a page that has transaction data we need to track.